package.json

```diff
@@ -82,5 +85,13 @@
   "homepage": "https://github.com/googleapis/gax-nodejs#readme",
   "engines": {
     "node": ">=6.0.0"
+  },
+  "nyc": {
+    "include": [
```

Comment:
So generally I don't have to include the include section :) We should also add `build/test` to the exclude section.

Reply:
I actually need to exclude more stuff if I don't use include :)  (changed to only excluding things)